packages/node/src/utils/cosmos.ts

```diff
@@ -132,6 +132,15 @@ export function filterMessageData(
             ? decodedMsgData.toNumber()
             : decodedMsgData.toString();
       }
+
+      if (typeof decodedMsgData === 'bigint') {
+        if (BigInt(filter.values[key]) === decodedMsgData) {
+          continue;
+        } else {
+          return false;
+        }
```

Comment:
```suggestion
        if (BigInt(filter.values[key]) 1== decodedMsgData) {
          return false;
        }
```

Reply:
The continue statement cannot be removed; otherwise, it will enter the next if statement where no type conversion is performed. It must return false.